.github/ISSUE_TEMPLATE/---bug.yml

```diff
@@ -0,0 +1,62 @@
+name: 🐛 Bug
+description: Report a bug.
+labels: ["status: needs triage", "type: bug"]
+body:
+  - type: checkboxes
+    id: readfirst
+    attributes:
+      label: READ BEFORE CREATING!
+      description: |
+        - Avoid creating duplicates! Read the FAQ page and search through Issues and Discussions before creating one.
+        - Try to make a simple but descriptive title, and include the detailed information below. This could save a lot of time and help us fix issues swiftly.
+        - Make sure to use the latest version of Scratch Addons.
+        - For help, check our [FAQ page](https://scratchaddons.com/faq) or ask on [Discussions](https://github.com/ScratchAddons/ScratchAddons/discussions) or our [Discord server](https://discord.gg/R5NBqwMjNc).
+      options:
+        - label: OK
+          required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Include a clear and concise description of what should happen and what's actually happening.
+      placeholder: I can't see the button...
+    validations:
+      required: true
+
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to reproduce
+      description: Provide an unambiguous set of steps to reproduce this bug.
+      placeholder: |
+        1. Open a Scratch project.
+        2. Click See inside.
+        3. Open the costume editor.
+      value: |
+        1.
+        2.
+        3.
+    validations:
+      required: true
+
+  - type: textarea
+    id: additonalcontext
+    attributes:
+      label: Demonstration(s) and additional context
+      description: If applicable, insert screenshots or a live example to help demonstrate your problem, as well as any other useful infomation here.
+    validations:
+      required: false
+
+  - type: textarea
```

Comment:
Please split this into 3 single line text boxes

Reply:
I was considering that, but wasn't going to do it initially because every field would have its own heading, like:
> ### Browser
> Chrome 89
> ### Scratch Addons version
> 1.10.0
> ### Operating System
> Windows

...right? Or is there a way to do it all in one heading thing while still using single line inputs?

Reply:
I thought it put label to the left of the input, but I'm not sure.

Reply:
> I thought it put label to the left of the input, but I'm not sure.

It doesn't.

Reply:
I disagree with Paul at this one, and I think this is fine.

Reply:
@DNin01 Would you mind merging the enviroment fields back?

Reply:
> @DNin01 Would you mind merging the enviroment fields back?

Will do.